[2024-07-14T04:21:20.305+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-14T04:21:20.578+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-07-10T00:00:00+00:00 [queued]>
[2024-07-14T04:21:20.708+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-07-10T00:00:00+00:00 [queued]>
[2024-07-14T04:21:20.720+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-14T04:21:21.524+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): transform.dim_employees.test> on 2024-07-10 00:00:00+00:00
[2024-07-14T04:21:21.583+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=16272) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-14T04:21:21.595+0000] {standard_task_runner.py:63} INFO - Started process 16290 to run task
[2024-07-14T04:21:21.593+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.dim_employees.test', 'scheduled__2024-07-10T00:00:00+00:00', '--job-id', '1283', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpztwhza3w']
[2024-07-14T04:21:21.625+0000] {standard_task_runner.py:91} INFO - Job 1283: Subtask transform.dim_employees.test
[2024-07-14T04:21:21.837+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-14T04:21:22.225+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-07-10T00:00:00+00:00 [running]> on host airflow-scheduler
[2024-07-14T04:21:23.220+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.dim_employees.test' AIRFLOW_CTX_EXECUTION_DATE='2024-07-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-10T00:00:00+00:00'
[2024-07-14T04:21:23.225+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-14T04:21:23.229+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-07-14T04:21:23.243+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmp49zje7y5 from /opt/airflow/dags/dbt/dbt_project
[2024-07-14T04:21:23.293+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-07-14T04:21:23.674+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-07-14T04:21:23.696+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-07-14T04:21:23.716+0000] {config.py:298} INFO - Profile caching is enable.
[2024-07-14T04:21:23.726+0000] {config.py:272} INFO - Profile found in cache using profile: /tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad/profiles.yml.
[2024-07-14T04:21:23.757+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmp49zje7y5', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmp49zje7y5
[2024-07-14T04:21:23.775+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:21:23.778+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmp49zje7y5', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:21:23.895+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:21:38.892+0000] {subprocess.py:94} INFO - [0m04:21:38  Running with dbt=1.8.3
[2024-07-14T04:21:40.781+0000] {subprocess.py:94} INFO - [0m04:21:40  Updating lock file in file path: /tmp/tmp49zje7y5/package-lock.yml
[2024-07-14T04:21:40.829+0000] {subprocess.py:94} INFO - [0m04:21:40  Installing dbt-labs/dbt_utils
[2024-07-14T04:21:42.107+0000] {subprocess.py:94} INFO - [0m04:21:42  Installed from version 0.8.6
[2024-07-14T04:21:42.118+0000] {subprocess.py:94} INFO - [0m04:21:42  Updated version available: 1.2.0
[2024-07-14T04:21:42.159+0000] {subprocess.py:94} INFO - [0m04:21:42
[2024-07-14T04:21:42.187+0000] {subprocess.py:94} INFO - [0m04:21:42  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-07-14T04:21:42.195+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:21:47.220+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:21:47.223+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:21:47.225+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'dim_employees', '--project-dir', '/tmp/tmp49zje7y5', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmp49zje7y5
[2024-07-14T04:21:47.229+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:21:47.233+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'dim_employees', '--project-dir', '/tmp/tmp49zje7y5', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:21:47.302+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:22:00.395+0000] {subprocess.py:94} INFO - [0m04:22:00  Running with dbt=1.8.3
[2024-07-14T04:22:03.238+0000] {subprocess.py:94} INFO - [0m04:22:03  Registered adapter: snowflake=1.8.3
[2024-07-14T04:22:08.414+0000] {subprocess.py:94} INFO - [0m04:22:08  Found 20 models, 27 data tests, 11 sources, 684 macros
[2024-07-14T04:22:08.431+0000] {subprocess.py:94} INFO - [0m04:22:08
[2024-07-14T04:22:15.999+0000] {subprocess.py:94} INFO - [0m04:22:15  Concurrency: 1 threads (target='dev')
[2024-07-14T04:22:16.002+0000] {subprocess.py:94} INFO - [0m04:22:16
[2024-07-14T04:22:16.138+0000] {subprocess.py:94} INFO - [0m04:22:16  1 of 2 START test not_null_dim_employees_EMPLOYEEID ............................ [RUN]
[2024-07-14T04:22:18.184+0000] {subprocess.py:94} INFO - [0m04:22:18  1 of 2 PASS not_null_dim_employees_EMPLOYEEID .................................. [[32mPASS[0m in 2.01s]
[2024-07-14T04:22:18.239+0000] {subprocess.py:94} INFO - [0m04:22:18  2 of 2 START test unique_dim_employees_EMPLOYEEID .............................. [RUN]
[2024-07-14T04:22:20.164+0000] {subprocess.py:94} INFO - [0m04:22:20  2 of 2 PASS unique_dim_employees_EMPLOYEEID .................................... [[32mPASS[0m in 1.93s]
[2024-07-14T04:22:20.186+0000] {subprocess.py:94} INFO - [0m04:22:20
[2024-07-14T04:22:20.187+0000] {subprocess.py:94} INFO - [0m04:22:20  Finished running 2 data tests in 0 hours 0 minutes and 11.74 seconds (11.74s).
[2024-07-14T04:22:20.929+0000] {subprocess.py:94} INFO - [0m04:22:20
[2024-07-14T04:22:20.932+0000] {subprocess.py:94} INFO - [0m04:22:20  [32mCompleted successfully[0m
[2024-07-14T04:22:20.936+0000] {subprocess.py:94} INFO - [0m04:22:20
[2024-07-14T04:22:20.947+0000] {subprocess.py:94} INFO - [0m04:22:20  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:22:23.934+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:22:23.938+0000] {local.py:254} INFO - [0m04:22:20  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:22:24.967+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-07-14T04:22:24.993+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-07-14T04:22:25.009+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://OU00370.europe-west2.gcp/ELT_BATCH.dwh.dim_employees', extra=None)]
[2024-07-14T04:22:25.035+0000] {local.py:350} INFO - Outlets: []
[2024-07-14T04:22:25.038+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-14T04:22:25.440+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-14T04:22:25.643+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.dim_employees.test, run_id=scheduled__2024-07-10T00:00:00+00:00, execution_date=20240710T000000, start_date=20240714T042120, end_date=20240714T042225
[2024-07-14T04:22:25.737+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-14T04:22:25.798+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
